osidb/models.py

```diff
@@ -828,8 +853,12 @@ def _validate_summary_major_incident(self):
         Check that a flaw that is a major incident has a summary
         """
         req = self.meta.filter(type=FlawMeta.FlawMetaType.REQUIRES_SUMMARY).last()
-        if not self.is_major_incident_temp() or (
-            req and req.meta_attr.get("status") == "-"
+        if (
```

Comment:
FYI: `_validate_summary_major_incident`'s behaviour is not fully correct (e.g. MI flaw without `summary` and without `requires_summary` should raise an alert). I will update it during the work on OSIDB-1104.